### PR TITLE
Allow specifying derivation path on account creation

### PR DESCRIPTION
**Pull request by vanruch**

### CODE REVIEW

**Review by barrutko — COMMENTED** [review]
IMO looks good. But let's wait for @krzysztof-jelski's review.

**Review by vanruch — COMMENTED** [review]
Looks good for me

File: packages/extension-ui/src/Popup/CreateAccount/CreateAccount.test.tsx [hunk 1]
```diff
@@ -98,46 +108,83 @@ describe('Create Account', () => {
...
+    it('derivation path field shows error when derivation path is invalid', async () => {
+      jest.spyOn(messaging, 'validateSeed').mockRejectedValue('');
+      fillInputs('abc', 'abcdef', 'abcdef');
+      wrapper.find(DerivationPath).find(OptionsLabel).simulate('click');
+      type(wrapper.find(DerivationPath).find('input'), 'invalid-path');
+      await act(flushAllPromises);
+      wrapper.update();
+
+      expect(wrapper.find(DerivationPath).find(InputWithLabel).prop('isError')).toBe(true);
+    });
```
Comment (krzysztof-jelski) [inline packages/extension-ui/src/Popup/CreateAccount/CreateAccount.test.tsx]: Remove duplication between those 2 tests

File: packages/extension-ui/src/Popup/CreateAccount/DerivationPath.tsx [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+    pairType: KeypairType;
+    path: string;
+  }) => void;
+  onExpand: (reverse: boolean) => void;
+  isValid: boolean;
+}
+
+const keypairs = {
+  ed25519: {
+    name: 'Edwards (ed25519)',
```
Comment (krzysztof-jelski) [inline packages/extension-ui/src/Popup/CreateAccount/DerivationPath.tsx:23]: Maybe we could import the possible types from the libs?

**Review by krzysztof-jelski — APPROVED** [review]
Check import cases.

**Pull request merged**